bin/inference/pycbc_inference

```diff
@@ -208,12 +209,16 @@ with ctx:
     generator_function = generator.select_waveform_generator(
                                                     static_args["approximant"])
 
+    # get gates for templates
+    gates = gate.gates_from_cli(opts)
```

Comment:
Since you removed the other ``logging`` message, can you add like a something like even a simple `` logging.info("Gating {} strain")`` here? I think its would be good to keep that on the terminal somewhere.

Reply:
@cmbiwer This point just loads the gates from the config file. There is still a message in `data_from_cli`; see lines [377](https://github.com/ligo-cbc/pycbc/blob/master/pycbc/inference/option_utils.py#L377) and [420](https://github.com/ligo-cbc/pycbc/blob/master/pycbc/inference/option_utils.py#L420) of `option_utils.py`. I remove the message from the gate function because it tried printing out to screen for every single iteration.